### ISSUE TITLE
reproduced visualization results

### ISSUE THREAD

**Issue by twowwj** [zycliao/skeleton-free-pose-transfer#7]
Hi,

I followed the code for the division of the amass dataset. But found the visualization to be very poor. 
There were some bugs in the training code and I tried to fix them.
![image](https://user-images.githubusercontent.com/33343889/189162734-5809c97d-f269-4575-8e18-7fdff3b7dae4.png)
![image](https://user-images.githubusercontent.com/33343889/189162762-fc12ab44-5354-4533-9bdb-531698357470.png)
Is there something I've missed?
amass_splits = {
    'train': ['BMLmovi', 'KIT', 'EKUT', 'TotalCapture', 'Eyes_Japan_Dataset', 'ACCAD', 'CMU'],
    'test': ['HumanEva', 'SFU', 'Transitions_mocap', 'SSM_synced', 'DanceDB'],
}
Looking forward to your reply!

thx~

**Comment by twowwj** [comment]
https://github.com/zycliao/skeleton-free-pose-transfer/blob/ce04cac9d911131900e410dba8642bdee09c3f4b/train.py#L59
I set the "simplify=False"
I suspect this is because I didn't simplify the amass data and kept 6890 vertices to train, could this be the main reason for the poor visualization results during training?

**Comment by zycliao** [comment]
I'm not sure what causes this. But `simplify` does not change the result much.

**Comment by twowwj** [comment]
![image](https://user-images.githubusercontent.com/33343889/191130599-72855efd-ea74-4943-a8b3-21b46d57fecf.png)
thanks for your reply!
I noticed during training that loss_sw_sim is always 0; is it normal that loss_sw is moving from -0 to the negative half-axis for the first few epochs, gradually getting smaller and finally becoming 0?

**Comment by zycliao** [comment]
loss_sw_sim is used to make the skinning weight before and after augmentation to be the same. Without augmentation, it's always 0.

loss_sw contains two kl divergence and one is negative. So it could have any sign. This is explained in the paper in detail. When both KL div are zero, it means for sampled pairs of points, there's no error. This is normal when the sampled points' predicted skinning weights are rigid (like [1, 0, 0, 0])

**Comment by twowwj** [comment]
thanks for you reply!
